vmdb/lib/miq_automation_engine/models/miq_ae_class_copy.rb

```diff
@@ -78,15 +78,14 @@ def add_fields
 
   def validate
     dest_class = MiqAeClass.find_by_fqname("#{@target_ns_fqname}/#{@target_name}")
-    if dest_class
-      dest_class.destroy if @overwrite
-      raise "Destination Class already exists #{dest_class.fqname}" unless @overwrite
-    end
+    return unless dest_class
+    dest_class.destroy if @overwrite
```

Comment:
@mkanoor yea - I'm frustrated with the guard clause stuff in rubocop.
I prefer a single exit if possible.
I'll discuss w/ @Fryguy